readme.md

```diff
@@ -1 +1,54 @@
-# Hello World
+# Form Builder
+
+**Form Builder** is a simple node application that generates a form based on given inputs.
+
+<div align="center">
+
+​		<img src="/docs/demo.png" width="220">
+
+</div>
+
+## Getting Started
+
+### Prerequisites
+
+[Node](https://nodejs.org/en/)
+
+### Installing
+
+Clone Repo
+
+```
+$ git clone https://github.com/reactivepixel/ColabCode.git
```

Comment:
The "clone repo" command is grabbing the root master branch of the repo, rather than your specific branch.

Reply:
The code should not, without great need and consideration, reference a specific branch, as the branch model defines various use s for each branch and expects the code will migrate across multiple branches.